training/src/main/scala/com/airbnb/aerosolve/training/TrainingUtils.scala

```diff
@@ -258,6 +258,40 @@ object TrainingUtils {
       .collect
   }
 
+  def getLabelCounts(minCount : Int,
+                     input : RDD[Example],
+                     rankKey: String) : Array[((String, String), Int)] = {
```

Comment:
`rankKey` is the featureFamily name for label, we are using this word many places in Aerosolve. I feel we can keep it this way in this PR. Perhaps do a refactor in the future if necessary